pkg/mocks/networking.go

```diff
@@ -35,6 +35,20 @@ func (m *MockNetworkAccess) EXPECT() *MockNetworkAccessMockRecorder {
 	return m.recorder
```

Comment:
May be good to have the mock in its own file, to not mix test code with prod code